bats/tests/helpers/os.bash

```diff
@@ -84,16 +84,29 @@ sudo_needs_password() {
 
 supports_vz_emulation() {
     if is_macos; then
+        if [[ -n ${_RD_SUPPORTS_VZ_EMULATION:-} ]] || load_var _RD_SUPPORTS_VZ_EMULATION; then
+            if is_true _RD_SUPPORTS_VZ_EMULATION; then
+                return 0
+            else
+                return 1
+            fi
+        fi
```

Comment:
This whole logic could be written

```bash
run is_true "${_RD_SUPPORTS_VZ_EMULATION:-}"
return "$status"
```

Reply:
Nice.

(I don't need the `:-` fallback because the `if` on line 87 means it's either non-empty, or we successfully restored the variable.)